Kitchen fstuff

chowdermcarthor opened the pull request:
*Minor changes to the kitchen/botany.
*Fixed microwave securing/unsecuring messages being switched.

chowdermcarthor committed:
Made quality of life changes to the kitchen and botany, fixed the microwave's securing/unsecuring words being flipped.

chowdermcarthor committed:
Merge branch 'dev' of https://github.com/Baystation12/Baystation12 into dev

PsiOmegaDelta commented:
Image(s) of the kitchen/botany adjustments?

PsiOmegaDelta commented on code/game/machinery/kitchen/microwave.dm:
```diff
@@ -137,12 +137,12 @@
 		return 1
 	else if(istype(O,/obj/item/weapon/crowbar))
 		user.visible_message( \
-			"<span class='notice'>\The [user] begins [src.anchored ? "securing" : "unsecuring"] the microwave.</span>", \
```
If the microwave is already anchored, I'm pretty sure you're not securing it even more. I need to look at this IG.

chowdermcarthor replied:
It was the opposite, I've dealt with this for several rounds.

PsiOmegaDelta replied:
Nevermind me, I read the wrong side of the diff.

chowdermcarthor commented:
https://i.gyazo.com/ae0f8d017c7dbdb5a9413fa3042feb56.png
Redid botany to be more open and eaiser to move through.
Moved most of the misc. cooking machines to the freezer.
Redid tables.
Moved the to the front so people could vend their own utensils.
Added some more microwaves.

babydoll commented:
Grills and ovens, instruments that rely on creating heat in order to prepare food, aren't traditionally placed in a walk in freezer, where the local atmosphere has been modified in order to eliminate heat.

chowdermcarthor commented:
Yeah well they take up space.

babydoll commented:
Well, it's not complicated.
![1487993643](https://cloud.githubusercontent.com/assets/8846073/23327843/b5749f62-fae1-11e6-85e7-8e247b1e66e5.png)

Take all the computers out of the bridge, put them in the hallway. They take up space.

chowdermcarthor committed:
no message

chowdermcarthor commented:
I don't feel like arguing I'll just move 'em.

Kasuobes commented:
Map needs cleaning.

chowdermcarthor commented:
I need to know specifics to do that.

Kasuobes commented:
https://baystation12.net/forums/threads/setting-up-using-the-maptools.10/

chowdermcarthor commented:
I meant specifics as in what needs changed.

Kasuobes commented:
Literally cleaning the map, torch-2.dmm.
